code/game/shuttle_engines.dm

```diff
@@ -155,6 +155,26 @@
 	bound_height = 96
 	appearance_flags = 0
 
+/obj/structure/shuttle/engine/hugeionengine
+	name = "engine"
```

Comment:
the name should be clearer and more flavorful, like "bluespace-ion engine"

Reply:
how is? "Nanotrasen MkIII BPDT engine"